samples/subsys/logging/logger/src/main.c

```diff
@@ -30,7 +30,7 @@ static struct k_mem_partition *app_parts[] = {
 #endif /* CONFIG_USERSPACE */
 
 /* size of stack area used by each thread */
-#define STACKSIZE 1024
+#define STACKSIZE (1024 + CONFIG_TEST_EXTRA_STACKSIZE)
```

Comment:
this looks a bit strange as define name suggests that it is test specific and yet it is used in a sample. Maybe stacksize should be bigger here?

Which platforms are more stack hungry? Maybe size should depend on whether it is 64 bit platform or not?

Reply:
it's one of the tests performed when using `sanitycheck` so it behaves like a test after all. I need a bigger stack size for the ARM64 port.

Reply:
it's likely that 64 bit platform needs more stack thus i thought that may it should be something like
```
#define STACK_SIZE (1024 + ((IS_ENABLED(CONFIG_64BIT) ? 1024 : 0))
```
That is kind of a hint for a user that in his application he also need to consider different stack sizes for 64 bit platforms. I am not sure how fair is the assumption that 32 bit platforms use more or less same stack.

Reply:
While I agree with you I wonder what's the actual difference between tests and samples since both are actually tested during the sanity check? In the tests we have `CONFIG_TEST_EXTRA_STACKSIZE` that, even though it is misnamed for samples, it is exactly doing what we want and need.

I'm fine with both the solutions but I just think that we already have what we need, just with the wrong name maybe.

Reply:
@nordic-krch we typically leave this to the ARCHitecture, i.e. define the EXTRA Stack Size they need.

Reply:
We do similar things in other tests/samples